docs/features/routing.rst

```diff
@@ -2,7 +2,7 @@ Routing
 =======
 
 Ocelot's primary functionality is to take incoming http requests and forward them on to a downstream service. Ocelot currently only supports this in the form of another http request (in the future
-this could be any transport mechanism). 
+this could be any transport mechanism).

```

Comment:
I recommend to switch to Visual Studio instead of Visual Code.
We have default VS IDE and StyleCop analyzers, and some other Git rules, so it is better to develop in Visual Studio, please!
What style checkers or linters do you use in Visual Code?
At least you need to switch off them!

Reply:
Great!
I have fixed the issue in https://github.com/ThreeMammals/Ocelot/pull/1684/commits/6d29040db6570a01847ef596a2cdc583e146ad6c just for 5th line, but we have the same issue on lines: 33, 37, 41, 70, 94, 111, 136, 163, 177 and 179 !!!
When I said "there is white space issue on line 5", I meant all the same issues in the same file!

Why don't remove the diff of these lines too?

Reply:
Missed those. Let me get back to those. :+1: